xtask/src/release.rs

```diff
@@ -99,7 +99,7 @@ impl PromoteCmd {
         cmd!("git add src/tools/rust-analyzer").run()?;
         cmd!("git commit -m':arrow_up: rust-analyzer'").run()?;
         if !self.dry_run {
-            cmd!("git push").run()?;
+            cmd!("git push origin {branch}").run()?;
```

Comment:
Not everyone necessarily calls it this way. I for example use origin as the upstream repo and my as my fork for almost all repos. I don't think it is a big deal though if everyone that does the sync uses the same name for their fork.

Reply:
I use `origin` and `mine`, but this is implied by [the code above](https://github.com/rust-analyzer/rust-analyzer/pull/7597/files#diff-aca9c10b837027d07a847e6843b0088f73eb46f59d2c79cda87fd067d2a924ffR87-R98) and documented in the [release docs](https://github.com/rust-analyzer/rust-analyzer/tree/master/docs/dev#release-process).

Reply:
Yeah I noticed this too, and think it would be good to fix at some point.

I think this PR would cause me to push branches to `rust-lang/rust` directly, which I'd rather not do.